shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/main/java/org/apache/shardingsphere/data/pipeline/core/spi/ratelimit/TPSJobRateLimitAlgorithm.java

```diff
@@ -56,13 +56,8 @@ public String getType() {
     
     @Override
     public void intercept(final JobOperationType type, final Number data) {
-        switch (type) {
-            case INSERT:
-            case DELETE:
-            case UPDATE:
-                break;
-            default:
-                return;
+        if (JobOperationType.SELECT == type) {
+            return;
         }
```

Comment:
It's better to keep INSERT/DELETE/UPDATE, since JobOperationType might add new type, then it might cause issue